Description: Restores the backward compatibility of PropertyUtils
 by removing the IOException on the loadProperties methods
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/src/main/java/org/codehaus/plexus/util/PropertyUtils.java
+++ b/src/main/java/org/codehaus/plexus/util/PropertyUtils.java
@@ -35,19 +35,30 @@
 {
 
     public static Properties loadProperties( final URL url )
-        throws IOException
     {
-        return loadProperties( Objects.requireNonNull( url, "url" ).openStream() );
+        try
+        {
+            return loadProperties( Objects.requireNonNull( url, "url" ).openStream() );
+        }
+        catch ( IOException e )
+        {
+            throw new RuntimeException(e);
+        }
     }
 
     public static Properties loadProperties( final File file )
-        throws IOException
     {
-        return loadProperties( Files.newInputStream( Objects.requireNonNull( file, "file" ).toPath() ) );
+        try
+        {
+            return loadProperties( Files.newInputStream( Objects.requireNonNull( file, "file" ).toPath() ) );
+        }
+        catch ( IOException e )
+        {
+            throw new RuntimeException(e);
+        }
     }
 
     public static Properties loadProperties( final InputStream is )
-        throws IOException
     {
         final Properties properties = new Properties();
         
@@ -58,6 +69,10 @@
             {
                 properties.load( in );
             }
+            catch ( IOException e )
+            {
+                throw new RuntimeException(e);
+            }
         }
 
         return properties;
